lessons/pydata/homework_revisited/index.ipynb

```diff
@@ -0,0 +1,370 @@
+{
```

Comment:
<p>načtĚte...</p><p> </p><p>one hot encoding - nepoužije to žádnou třídu, která by se tak jmenovala, ale prostě metodu ;-)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>